Makefile for this test case does not work with parallel build.

--- ghc-9.4.4.orig/testsuite/tests/driver/T21035/Makefile	2023-03-06 18:50:58.614955037 +0100
+++ ghc-9.4.4/testsuite/tests/driver/T21035/Makefile	2023-03-06 19:20:09.027840193 +0100
@@ -41,3 +41,6 @@ T21035: a.out
 clean:
 	find . \( -name "*.*hi" -o -name "*.*o" \) -delete
 	rm -rf hsdep/pkgdb hsdep-empty-lib a.out
+
+# This Makefile does not reliably work with parrallel Make.
+.NOTPARALLEL:
